projects/ion/src/lib/checkbox/checkbox.component.spec.ts

```diff
@@ -133,7 +138,9 @@ describe('CheckboxComponent', () => {
       });
       const element = screen.getByTestId(boxId);
       fireEvent.click(element);
-      expect(clickEvent).toHaveBeenLastCalledWith(StateEvents[state]);
+      const expectedEmission = StateEvents[state];
+      expectedEmission.value = '';
```

Comment:
should emit ' ' when is clicked in enabel checkbox?

Reply:
Yes. Every time it emit status, it'll emit value too. By default, value has an empty string ('').